needles/fbe_factory_dialog.json

```diff
@@ -12,6 +12,14 @@
       "height": 230,
       "type": "match"
     },
+    {
+      "": "Release version string",
```

Comment:
I wrote exactly this patch (± a few pixels) last week, but apparently didn't push it and open the PR :(

Reply:
Sad patch panda.